dataland-frontend/src/components/forms/parts/elements/basic/SingleSelectFormElement.vue

```diff
@@ -11,38 +11,13 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from "vue";
+import { ComponentPropsOptions, defineComponent } from "vue";
 import { FormKit } from "@formkit/vue";
-import { DropdownOption } from "@/utils/PremadeDropdownDatasets";
+import { DropdownOptionFormFieldProps } from "@/components/forms/parts/fields/FormFieldProps";
 
```

Comment:
The component `SingleSelectFormField`, which mounts a `SingleSlectFormElement`, passed the hardcoded value `"short"` as `inner-class` prop to `SingleSlectFormElement`.
This is not necessary, since `"short"` is the default value, if you don't pass a value for the prop `inner-class`.

Reply:
I don't see here what you say. Please explain.

Reply:
yes, right

Reply:
👍    Please notify me as soon as this has been taken care of.

Reply:
done

Reply:
👍